developer-docs/installation/content_service.md

```diff
@@ -7,10 +7,69 @@ allowSearch: true
 ---
 
 ## Overview
+This Document is created to help the developers to set-up content service in local linux machine. So Developer can access the content stored in the remote repositories.
 
+## System Requirements
 
+To install Sunbird Content Service, ensure that your laptop or desktop has the following minimum system requirements:
 
-## Prerequisites
+- Operating System: Linux  
+- RAM: 2GB or more
+- CPU: 1 cores, 2 GHz or more
 
-## Installment
+### Pre-requisites
 
+1. [Install node](https://nodejs.org/en/download/)
+2. [Install Imagemagick](https://www.npmjs.com/package/gm)
+3. [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+### Environment Variables:
+|                Variable                      |               Value                |
+| ---------------------------------------|:----------------------------------:|
+| sunbird_content_provider_api_base_url  | content provider API base url. e.g.: https://qa.ekstep.in or https://api.ekstep.in | 
+| sunbird_content_provider_api_key       |  API key for the above content provider URL please refer http://docs.sunbird.org/latest/developer-docs/telemetry/authtokengenerator_jslibrary/#how-to-generate-authorization-credentials for api keys. |
```

Comment:
@iostream04 Pls make these URLs relative. So that later if version changes it wont break urls. Here Relative url should be ```developer-docs/telemetry/authtokengenerator_jslibrary/#how-to-generate-authorization-credentials```